x-pack/plugins/index_lifecycle_management/public/store/middleware/set_selected_policy_from_selected_template.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  fetchedIndexTemplate,
+  fetchPolicies,
+  setSelectedPolicy,
+  setPhaseData
+} from '../actions';
+import { getSelectedNodeAttrs, getPhaseData } from '../selectors';
+import { PHASE_WARM, PHASE_NODE_ATTRS, PHASE_COLD } from '../constants';
+
+export const setSelectedPolicyFromSelectedTemplate = store => next => async action => {
+  if (action.type === fetchedIndexTemplate().type) {
+    const template = action.payload;
+    if (template.settings.index && template.settings.index.lifecycle) {
```

Comment:
What's the best way to set up an index to test this branch of the logic?

Reply:
You can go through the wizard once and save, and then reload the wizard and select the same template as for your first pass.